packages/wonder-blocks-button/components/button.md

```diff
@@ -537,7 +537,7 @@ class Example extends React.Component {
     }
 
     componentWillUnmount() {
-        this.timeout.clear();
+        this.timeout && this.timeout.clear();
```

Comment:
Can't wait until something like ADR-77 lands and we can have a standard way to handle all this.